ui/.env.gcr

```
@@ -1 +1,5 @@
-VITE_API_BASE_URL=https://comicchase-service-664556590362.us-central1.run.app/api
+# Firebase Hosting + Cloud Run 環境配置
+# Cloud Run 使用自訂網域 api.comicchase.site
+# 前端和後端共用同一根網域，避免 CORS 問題
+VITE_API_BASE_URL=https://api.comicchase.site
+VITE_ALLAUTH_BASE_URL=https://api.comicchase.site/_allauth/browser/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ALLAUTH_BASE_URL key should go before the VITE_API_BASE_URL key

(UnorderedKey)